src/createServer.js

```diff
@@ -1,10 +1,47 @@
+/* eslint-disable max-len */
 'use strict';
 
+/* eslint-disable node/no-deprecated-api */
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const absolutePaths = path.resolve(url.pathname);
+
+    res.setHeader('Content-Type', 'text/plain');
+
+    if (!absolutePaths.includes('file')) {
```

Comment:
The condition `!absolutePaths.includes('file')` is misleading. It seems like you want to check if the path includes 'file', but the logic should be based on the URL path, not the resolved absolute path. Consider revising this logic.